Item 8417: Show reference guideset to inform the selected date range

labkey-ankurj opened the pull request:
#### Rationale
If the date range(s) for the Guide Set(s) are too far apart from the selected dates, data from the dates in between will be truncated so that the Guide Set can be displayed together with the QC data from the custom dates.

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/2055

#### Changes
* New checkbox option to see reference guide set

labkey-ankurj committed:
Show out of range guidesets checkbox and lint function curleys

labkey-ankurj committed:
Determine first distant guideset in the past

labkey-ankurj committed:
Filter the data points between guideset end and custom date range start

labkey-ankurj committed:
Mask the region between the end of left guideset and start of 5 points

labkey-ankurj committed:
disconnect the qc graph b/w guideset and points of interest

labkey-ankurj committed:
fix edge case

labkey-ankurj committed:
change ui checkbox text

labkey-ankurj committed:
self code review

labkey-ankurj committed:
code review feedback

labkey-ankurj committed:
fix checkBox name

labkey-ankurj committed:
code review feedback and fix manual testing issues

labkey-ankurj committed:
Merge branch 'develop' into fb_OORangeGS_8417

labkey-ankurj committed:
cr feedback

labkey-ankurj committed:
reverse color that changed during verification

labkey-sweta committed:
Link QC And Experimental Folder - Automation test

labkey-ankurj committed:
fix automated test failure and a bug

labkey-sweta committed:
Link QC And Experimental Folder - Automation test

labkey-sweta committed:
Adding category and timeout for the test

labkey-ankurj committed:
Merge branch 'develop' into fb_OORangeGS_8417

labkey-ankurj committed:
debug failure

labkey-sweta committed:
Adding explicit delete projectHome as workaround for teamcity test timeout

labkey-sweta committed:
Merge remote-tracking branch 'origin/fb_OORangeGS_8417' into fb_OORangeGS_8417

labkey-sweta committed:
Changing the timeout to 8 for testing

labkey-ankurj committed:
add more logging for TC debugging

labkey-ankurj committed:
remove debugging logs, reduce timeout and add wait before guideset creation

labkey-ankurj committed:
add actual wait

labkey-ankurj committed:
increase test timeout

labkey-ankurj committed:
one more

labkey-ankurj committed:
use the api container helper to delete the project

labkey-ankurj committed:
Merge branch 'release21.3-SNAPSHOT' into 21.3_fb_OORangeGS_8417

labkey-jeckels reviewed (changes requested):
This change adds 75MB of sample files. I was hoping that we'd be able to edit one of the existing files instead of creating a whole separate copy. And why is one of the .sky.zip files 50MB? The starting file was 25MB.

labkey-sweta commented:
> This change adds 75MB of sample files. I was hoping that we'd be able to edit one of the existing files instead of creating a whole separate copy. And why is one of the .sky.zip files 50MB? The starting file was 25MB.

Good catch josh, i had accidental zipped extra folder in SProCoPTutorial-QCFolderData.zip. I corrected that.

We created a new copy of new sample file because we were updating the acquired_time in the file and not to mess up other test using this file. In hindsight i should have selected the file which was smaller in size. 

Now only 25 Mb is additional sample file for QC folder.

Let me know if thats needs.

labkey-sweta committed:
Updates to sample files and test

1. Renamed SProCoPTutorial-ExperimentFolderData.zip to SProCoPTutorial.zip.
2. Deleted the original SProCoPTutorial.zip.
3. Created  SProCoPTutorial-QCFolderData.zip of 25 MB.
4. Updated the test to use new file name.

labkey-jeckels reviewed (approved):
This is OK for 21.3, but please keep it in mind going forward and maybe we can consolidate files to keep the footprint lean.

You may need to update the tooltip text that contains a filename that's not in use anymore. Good to merge if the test is passing.

labkey-sweta committed:
Updating the tooltip

labkey-sweta committed:
1. Corrected the QC sample file
2. Updated the tooltip file name
